portal-impl/src/com/liferay/portlet/internal/InvokerPortletImpl.java

```diff
@@ -195,15 +193,15 @@ public Portlet getPortlet() {
 
 	@Override
 	public ClassLoader getPortletClassLoader() {
-		ClassLoader classLoader =
-			(ClassLoader)_liferayPortletContext.getAttribute(
-				PluginContextListener.PLUGIN_CLASS_LOADER);
```

Comment:
@tinatian Does this mean that the {{PluginContextListener.PLUGIN_CLASS_LOADER}} constant is no longer referenced anywhere in the codebase? Does it serve a purpose anymore? Thanks, Neil

Reply:
@ngriffin7a I think we should clean it up, but for now there are still usages of it. I will put this into my list